.gitignore

```diff
@@ -87,3 +87,4 @@ htmlcov/
 .coverage
 # ansible-test coverage results
 test/units/.coverage.*
+/test/integration/cloud-config-azure.yml
```

Comment:
Curious what the Ansible team thinks of adding this to gitignore. Obviously this is something we don't want to commit. I'm curious what the git handling for `cloud-config-aws.yml` is. If this is not desired, let me know and I can remove this change from the PR. I guess the alternative is adding this global gitignore.